hack/go-install.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
+1 I think it makes sense to start using this script like in core CAPI and moving the tools (in follow-up PRs) out of the checked-in module